wagtailinventory/tests/test_views.py

```diff
@@ -1,8 +1,8 @@
 from django.core.management import call_command
 from django.test import TestCase
 from django.urls import reverse
-from django.utils.six.moves.urllib.parse import urlencode
 
+from six.moves.urllib.parse import urlencode
```

Comment:
We don't need to use `six` here because we no longer support Python 2.
```suggestion
from urllib.parse import urlencode
```

Reply:
Done